gateway.c

```diff
@@ -604,6 +606,8 @@ void LoadConfigFile()
 	ReadString(fp, "ftpPassword", Config.ftpPassword, sizeof(Config.ftpPassword), 0);
 	ReadString(fp, "ftpFolder", Config.ftpFolder, sizeof(Config.ftpFolder), 0);	
 
+	ReadBoolean(fp, "EnableKML", 0, &Config.EnableKML);

```

Comment:
This configuration entry is only for enabling KML output (like Habitat output which is a boolean too), and in this current changeset the data which is exported in the kml is not configurable (lat,lon and altitude). When new features for kml export will be implemented I suggest that we add configuration variable for these features, but since we don't know what may be supported in the future I suggest that we don't add configuration entries which are not supported. It would be nice to implement logging with the features you mentioned but this is out of scope of this kml feature.